tests/test_qb.py

```diff
@@ -1548,3 +1650,24 @@ def test_qb_enum_edit_02(self):
 
         self.assertEqual(e.color, default.Color.Violet)
         self.assertEqual(e.name, "red")
+
+    def test_qb_cast_scalar_01(self):
+        # scalar to scalar
+        from models.orm import std
+
+        result = self.client.get(std.str.cast(std.int64(1)))
+        self.assertEqual(result, "1")
+
+    def test_qb_cast_scalar_02(self):
+        # enum to scalar
+        from models.orm import default, std
+
+        result = self.client.get(std.str.cast(default.Color.Red))
+        self.assertEqual(result, "Red")
+
+    def test_qb_cast_scalar_03(self):
+        # scalar to enum
+        from models.orm import default, std
+
+        result = self.client.get(default.Color.cast(std.str("Red")))
```

Comment:
This is an example of where I feel the explicit `cast` is inconsistent.  We are casting a Python literal string `"Red"` to `std.str` via a (Pythonic) call to a constructor, whereas the second cast is via a classmethod.